### ISSUE TITLE
where is use of itr_forward

### ISSUE THREAD

**Issue by 123rc** [Multimodal-NER/RpBERT#6]
I have gone through the code didn't find where the itr_forward function was called

**Comment by MatrixCPU** [comment]
> 我浏览了代码没有找到调用 itr_forward 函数的位置

    def itr_forward(self, pairs: List[MyPair]):
        text_batch = [pair.sentence.text for pair in pairs]
        inputs = self.tokenizer(text_batch, padding=True, return_tensors='pt').to(self.device)
        outputs = self._bert_forward_with_image(inputs, pairs)
        feats = outputs.last_hidden_state[:, CLS_POS]
        logits = self.aux_head(feats)

        labels = torch.tensor([pair.label for pair in pairs], dtype=torch.long, device=self.device)
        loss = F.cross_entropy(logits, labels, reduction='mean')
        pred = torch.argmax(logits, dim=1).tolist()

        return loss, pred

**Comment by Cloriaaaaa12** [comment]
> 我检查了代码，没有找到 itr_forward 函数被调用的位置

hello，do you have the image dataset for text-img classification?
